feature-libs/customer-ticketing/components/customer-ticketing-details/customer-ticketing-details.component.html

```diff
@@ -0,0 +1,44 @@
+<ng-container *ngIf="ticketDetails$ | async as ticket">
```

Comment:
Just a reminder, the 'overview'  and 'chat' are not separate cms components? 

If yes, then it's fine having this customer-ticketing-details.component.html as I believe you'll just include statically the chat component in this tempalte right?

If not, then this file should be more decoupled in a way that this is only for the top part of cms component

Reply:
yes, overview and the chat are separate components.
SupportTicketDetailsComponent and SupportTicketUpdateComponent respectively.
Maybe we need to revise the names.